docs/Operators.md

```diff
@@ -6946,6 +6947,79 @@ expect(node, inputs=[x, s, bias], outputs=[y],
 </details>
 
 
+### <a name="Inverse"></a><a name="inverse">**Inverse**</a>
+
+  Calculates inverse of a square matrix or batches of square matrices.
+  Inverse takes one input tensor of shape `[*, M, M]`, where `*` is zero or more batch dimensions,
+  and the inner-most 2 dimensions form square matrices.
+  The output is a tensor of shape `[*, M, M]`, containing the individual inverses of all input submatrices.
```

Comment:
What happens when the matrix is rank-deficient, or more generally, ill-conditioned with a large condition number? We should clarify what the expectation is in that case.
I would recommend that we keep it flexible for backends and treat this as an undefined behavior. Any backend can choose detect the condition and throw exception, or simply return garbage result (possibly with a warning).

Reply:
Thanks for pointing this out. I agree that we should expect an exception from backend. Should I mention that rank-deficient matrices are not supported?

Reply:
I updated the spec noting that the op only supports fill-rank tensors.